sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java

```diff
@@ -12,6 +17,12 @@
  * @see DefaultAzureCredential
  */
 public class DefaultAzureCredentialBuilder extends CredentialBuilderBase<DefaultAzureCredentialBuilder> {
+    private boolean excludeEnvironmentCredential;
+    private boolean excludeManagedIdentityCredential;
+    private boolean excludeSharedTokenCacheCredential;
+    private boolean excludeAzureCliCredential;
```

Comment:
In .NET the interactive browser credential is also part of the DefaultAzureCredential but it is excluded by default.

Reply:
Yep, I'll be adding in separate PR and test it appropriately.